ocs_ci/ocs/resources/backingstore.py

```diff
@@ -197,13 +244,29 @@ def _create_backingstore(method, uls_dict):
                             cmdMap[method.lower()][cloud.lower()](
                                 cld_mgr, backingstore_name, uls_name, region
                             )
-                        # Todo: Raise an exception in case the BS wasn't created
+                        wait_for_resource_state(
+                            OCP(
+                                kind="backingstore",
+                                namespace=config.ENV_DATA["cluster_namespace"],
+                            ),
+                            constants.STATUS_READY,
```

Comment:
there are two healthy statuses for backing store, I've created this constant `HEALTHY_PV_BS` for the pv backing store but maybe we should change its name to `HEALTHY_BS` and use it here